### ISSUE TITLE
Bandage BD just won't install Could not find DiscordCanary.exe! AND Could not find DiscordPTB.exe!

### ISSUE THREAD

**Issue by Davth** [Jiiks/BetterDiscordApp#879]
how is this "better discord" when the damn thing won't even install, I've spend hours googling and NO ONE ELSE has this problem but me, and there's literally not a damn thing I can do more to try and fix it
if anybody can actually fix this they are god and I will suck your dick/clit
![bandagedbd_windows_2018-11-17_09-26-19](https://user-images.githubusercontent.com/33887693/48662157-ec238e00-ea4b-11e8-9428-686231cdd75a.png)

**Comment by rauenzi** [comment]
You have weird scaling on your PC which is hiding the checkboxes.

**Comment by Davth** [comment]
> 
> 
> You have weird scaling on your PC which is hiding the checkboxes.

I don't really, it's just THIS window, and I highly doubt that effects installed programs

**Comment by InfernalUnity** [comment]
Had the same problem, Try clicking on the words "able"  or the cut off word at the stop. Or you could download exactly what it's asking for the links are here - https://discordia.me/canary then https://discordia.me/ptb . After that once they finish downloading it just close both of them and run Better Discord again and it should show that both of those are downloaded.

**Comment by Davth** [comment]
> 
> 
> Had the same problem, Try clicking on the words "able" or the cut off word at the top. Or you could download exactly what it's asking for the links are here - https://discordia.me/canary then https://discordia.me/ptb . After that once they finish downloading it just close both of them and run Better Discord again and it should show that both of those are downloaded.

Yep tried that too, even when I got them both to light up, the install button is still greyed out

**Comment by aaronlink127** [comment]
Try opening Display settings and changing "Scale and layout" to 100%

**Comment by lyn90** [comment]
I had same problem and  the install button simply isn't clickable just as the OP  said.I found old  downloaded  Bandagedbd and installed,worked for me .It's the 0.3.0

**Comment by lyn90** [comment]
also I had that scaling thing that is shown in the screenie

**Comment by lyn90** [comment]
[https://drive.google.com/open?id=19XC2wg3SjQUzHe3o4QKCieoeYPFiAhh3](url)
if you wanna try

**Comment by RedCriterion** [comment]
same problem, button greyed out, even after installing canary and ptb

**Comment by RedCriterion** [comment]
atleast the old installer works, thanks lyn

**Comment by cmgjol010** [comment]
Click this text
[https://i.imgur.com/Bx2Pnaw.png](url)

**Comment by cmgjol010** [comment]
Click this text
https://i.imgur.com/Bx2Pnaw.png

**Comment by KRAYplayZz** [comment]
Same problem :(

**Comment by MegaSqueeze** [comment]
tick the ptb and canary boxes next to them and then install

**Comment by Davth** [comment]
okay just so everyone knows, these solutions no longer work as BD has had many updates, and when I went to try to install it again everything seemed to work fine, except that discord seemed to run as normal instead of being BD, I've more or less given up

**Comment by TazeDev** [comment]
> okay just so everyone knows, these solutions no longer work as BD has had many updates, and when I went to try to install it again everything seemed to work fine, except that discord seemed to run as normal instead of being BD, I've more or less given up

same thing happend here

**Comment by Kleveen** [comment]
BandageBD can't find Discord.Exe alone what I can do ?

**Comment by Kleveen** [comment]
Help I need BetterDiscord

**Comment by DevilRobo** [comment]
I am Also having that same problem Can anyone help me 
Plz

**Comment by DevilRobo** [comment]
Got the problem Fixed  Just enable **Install to Stable** Option  The IMG is here:-https://imgur.com/a/eLJ8Wvl
Then the option for installing will come

**Comment by FC-Strat** [comment]
I am on a mac, and I don't know anything about this, and I really, really need help.  Whenever I reinstall it, and click Instal Bandaged BD, there is nothing in the stable or anything, and I don't know what to type in, since whenever I press "Browse", it won't work. It is really stressing me out, and I really need help T_T
<img width="790" alt="Screen Shot 2021-03-24 at 6 29 57 PM" src="https://user-images.githubusercontent.com/81335009/112397067-578e6200-8ccf-11eb-91b7-2327011ccbcd.png">

**Comment by erik1102933** [comment]
Same with me. I'm on a Mac and BetterDiscord is just doing nothing and not installing onto my discord. I had it already but I realized that it was gone from discord so I tried to reinstall it but it just didn't work. Even uninstalling doesnt work so I'm pretty confused cause I also cant click browse and everytime i click stable ptn or canary it says, Failed to install to : ReferenceError: Cannot access 'fs' before initialization

**Comment by ecazilic** [comment]
> I am on a mac, and I don't know anything about this, and I really, really need help. Whenever I reinstall it, and click Install Bandaged BD, there is nothing in the stable or anything, and I don't know what to type in, since whenever I press "Browse", it won't work. It is really stressing me out, and I really need help T_T
> <img alt="Screen Shot 2021-03-24 at 6 29 57 PM" width="790" src="https://user-images.githubusercontent.com/81335009/112397067-578e6200-8ccf-11eb-91b7-2327011ccbcd.png">

heree just saw this on their discord server
![Uploading Screen Shot 2021-03-25 at 4.23.52 PM.png…]()

**Comment by ecazilic** [comment]
![Uploading Screen Shot 2021-03-25 at 4.23.52 PM.png…]()

**Comment by DevilRobo** [comment]
Don't Know anything about Mac Sory 
But I know that *betterDiscord* is against DiscordTos So I am not using it anymore

**Comment by gazergazer69** [comment]
Yup, I'm on MAC & I'm having the exact same issue. It used to work for me.. I uninstalled it a couple weeks ago and decided to install it again but now it's not working. Also, can someone create a discord invite for the BetterDiscord server with unlimited uses and post it here for us who are trying to get support please. I can't seem to find their discord server either.

<img width="799" alt="Screen Shot 2021-04-01 at 12 14 11 AM" src="https://user-images.githubusercontent.com/81739962/113257291-3f928180-927f-11eb-9d6e-7d3ab42a8af2.png">

**Comment by Kai19933** [comment]
I'm having the same problem too
